tests/unit/src/kakarot/instructions/test_system_operations.cairo

```diff
@@ -102,7 +102,8 @@ func test__exec_call__should_return_a_new_context_based_on_calling_ctx_stack{
     assert sub_ctx.return_data_len = ret_size.low;
     assert [sub_ctx.return_data] = ret_offset.low;
     assert sub_ctx.gas_used = 0;
-    assert sub_ctx.gas_limit = 0;
+    let (gas_felt, _) = Helpers.div_rem(Constants.TRANSACTION_GAS_LIMIT, 64);
+    assert_le_felt(sub_ctx.gas_limit, gas_felt);
```

Comment:
not meaningful here but fyi when it's possible it's better (more optimized) to use `assert_le` instead of `assert_le_felt`